krm-functions/gen-configmap-fn/go.mod

```diff
@@ -44,6 +44,6 @@ require (
 	k8s.io/kube-openapi v0.0.0-20230525220651-2546d827e515 // indirect
 	k8s.io/utils v0.0.0-20230505201702-9f6742963106 // indirect
 	sigs.k8s.io/json v0.0.0-20221116044647-bc3834ca7abd // indirect
-	sigs.k8s.io/kustomize/kyaml v0.14.2 // indirect
+	sigs.k8s.io/kustomize/kyaml v0.14.2
```

Comment:
this change is weird, this is generated code.

Reply:
vscode's linter moaned about it, I'll revert this change...